packages/react/src/themes/teams/components/Button/buttonVariables.ts

```diff
@@ -95,6 +106,17 @@ export default (siteVars: any): ButtonVariables => ({
   primaryBackgroundColorDisabled: siteVars.colorScheme.default.backgroundDisabled,
   primaryBorderColor: 'transparent',
 
+  invertedColor: siteVars.colors.white,
```

Comment:
Is the reason we need all these same colors to override other parts of the button?